wazo_dird/plugins/csv_backend/plugin.py

```diff
@@ -34,6 +35,10 @@ class CSVPlugin(BaseSourcePlugin):
     The `searched_columns` are the columns used to search for a term
     """
 
+    def __init__(self) -> None:
```

Comment:
We do not use type annotations in Wazo at this time. Not that I have anything against it but it's a change that should be discussed before using them.
```suggestion
    def __init__(self):
```

Reply:
Alternatively, the `_csv_last_modification_time` initialisation could also be moved to the `load` function